source/take-payments-by-phone-or-post.html.erb

```diff
@@ -61,7 +16,7 @@ title: Take payments by phone or post
           <a class="govuk-link" href="https://docs.payments.service.gov.uk/moto_payments/#take-a-payment-over-the-phone-moto-payments">take
             credit and debit card payments by telephone, or post</a>. These are sometimes called ‘MOTO’ payments.</p>
 
-        <p class="govuk-body">You get all of the advantages and security of using GOV.UK Pay, but you fill in your
+        <p class="govuk-body">You get all the advantages and security of using GOV.UK Pay, but you fill in your
```

Comment:
Let's keep this content the same please

Reply:
I'll revert but this reads strangely and should be looked at